src/main/java/hudson/plugins/ec2/EC2AbstractSlave.java

```diff
@@ -348,9 +355,16 @@ void stop() {
             LOGGER.info("EC2 instance stop request sent for " + getInstanceId());
             toComputer().disconnect(null);
         } catch (AmazonClientException e) {
-            Instance i = CloudHelper.getInstance(getInstanceId(), getCloud());
-            LOGGER.log(Level.WARNING, "Failed to stop EC2 instance: " + getInstanceId() + " info: "
-                    + ((i != null) ? i : ""), e);
+            Instance i = null;
+            try {
```

Comment:
I thought that I replaced everywhere the call, but it seems is not.
I am not a fan of the try/Catch in a try/Catch, can you find a different solution ? (call the public getInstance)